Makefile

```diff
@@ -155,7 +155,7 @@ gen-test-golden: test-unit
 # without a go.mod file when downloading the following dependencies
 
 $(GOX):
-	(cd /; GO111MODULE=on go install github.com/mitchellh/gox@latest)
+	(cd /; GO111MODULE=on go install github.com/mitchellh/gox@master)
```

Comment:
We've recently taken to pinning dependencies like this. Instead of master can you pin to the latest commit.

```
go install github.com/mitchellh/gox@v1.0.2-0.20220701044238-9f712387e2d2
```

Reply:
Ack. Fixed.